pagopa/pn-frontend
# PN-12800 - Handling virtual keys for PG users

## Short description
Handling virtual keys for PG users

## List of changes proposed in this pull request
- Create VirtualKeys section

## How to test
Start PG and go to "Integrazione API" section. You should see the VirtualKeys section
